### PR TITLE
path fixes

### PR DESCRIPTION
**Pull request by FinalAngel**
- remove reference to ``select2.png`` in ``_select2.scss``
- fix automatic path to glyphicons (requires aldryn-bootstrap3)

### CODE REVIEW

File: private/sass/libs/_select2.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
...0;}.select2-container-multi .select2-search-choice-close{left:3px}.select2-container-multi .select2-choices .select2-search-choice .select2-search-choice-close:hover,.select2-container-multi .select2-choices .select2-search-choice-focus .select2-search-choice-close{background-position:right -11px}.select2-container-multi.select2-container-disabled .select2-choices{background-color:#f4f4f4;background-image:none;border:1px solid #ddd;cursor:default}.select2-container-multi.select2-container-disabled .select2-choices .select2-search-choice{padding:3px 5px;border:1px solid #ddd;background-image:none;background-color:#f4f4f4}.select2-container-multi.select2-container-disabled .select2-choices .select2-search-choice .select2-search-choice-close{display:none;background:0 0}.select2-result-selectable .select2-match,.select2-result-unselectable .select2-match{text-decoration:underline}.select2-offscreen,.select2-offscreen:focus{clip:rect(0 0 0 0);width:1px;height:1px;border:0;margin:0;padding:0;overflow:hidden;position:absolute;outline:0;left:0}.select2-display-none{display:none}.select2-measure-scrollbar{position:absolute;top:-10000px;left:-10000px;width:100px;height:100px;overflow:scroll}
```
Comment (vxsx) [inline private/sass/libs/_select2.scss]: it would be amazing to specify for which version we use the styles here

Comment (vxsx) [inline private/sass/libs/_select2.scss]: wait, did you just modify the original styles for select2?

Comment (FinalAngel) [inline private/sass/libs/_select2.scss]: yeah thats the original styles from select2, i could add a comment but dunno which version we used

Comment (vxsx) [inline private/sass/libs/_select2.scss]: i think better approach is to override them at the end of this file (or even a separate one, given that this one shouldn't be touched)

also, there is a reference to select2-spinner.gif that we also don't serve

Comment (GaretJax) [inline private/sass/libs/_select2.scss]: @FinalAngel, @vxsx is right, collectstatic chokes on select2-spinner.gif as well.

**Comment by vxsx** [comment]
:+1:

**Pull request merged**